wiki/Contributing-to-Oppia-Android.md

```diff
@@ -8,13 +8,18 @@ If you run into any problems along the way, we're here to help! Check out our [w
 
 ## Table of Contents
 
+- [How to find information on this wiki](#how-to-find-information-on-this-wiki)
 - [Onboarding instructions](#onboarding-instructions)
   - [Guidance on submitting a PR](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR)
   - [Developing your skills](https://github.com/oppia/oppia-android/wiki/Developing-skills)
 - [Important: Ongoing Bazel migration](#important-ongoing-bazel-migration)
 - [Installing the Oppia web app](#installing-the-oppia-web-app)
 - [Communication channels](#communication-channels)
 
+## How to find information on this wiki
```

Comment:
@BenHenning Thanks for review, Actually I only added this demo for wiki search in github. You can review this.
Apart from this I added only TOC in all wiki pages.

Reply:
Thanks! This seems reasonable. We may need to change it in a few months when GitHub's new search functionality exits beta, but it should work in the interim. :)

Reply:
Yes sure, We will change it.
Is I add new TODO issue for that?